src/main/groovy/me/champeau/gradle/JapicmpAbstractTask.groovy

```diff
@@ -93,9 +98,11 @@ abstract class JapicmpAbstractTask extends AbstractTask {
         def options = new Options()
         options.outputOnlyModifications = onlyModified
         options.setAccessModifier(AccessModifier.valueOf(accessModifier.toUpperCase()))
+        options.htmlOutputFile = GOptional.fromNullable(htmlOutputFile?.absolutePath)
```

Comment:
It is actually very infortunate that japicmp depends on Guava. I tend to think one should not use it because of it's large history of breaking changes. The fact that it leaks into the plugin is not really a good thing.